bindings/pyroot_experimental/cppyy/CPyCppyy/src/MemoryRegulator.cxx

```diff
@@ -244,3 +244,17 @@ PyObject* CPyCppyy::MemoryRegulator::RetrievePyObject(Cppyy::TCppObject_t cppobj
 
     return nullptr;
 }
+
+//-----------------------------------------------------------------------------
```

Comment:
To be contributed to upstream cppyy too.

https://bitbucket.org/wlav/cpycppyy/src/master/